flow/packet-headers/rsvp.yaml

```diff
@@ -0,0 +1,1222 @@
+            The Length MUST be atleast 4, and MUST be a multiple of 4.
+          $ref: '#/components/schemas/Flow.RSVPExplicitRoute.Length'
+          x-field-uid: 2
+        ipv4_address:
+          x-field-pattern:
+            description: >-
+              This IPv4 address is treated as a prefix based on the prefix length value below. 
+              Bits beyond the prefix are ignored on receipt and SHOULD be set to zero on transmission.
+            format: ipv4
+            default: 0.0.0.0
+            features: [count]
+          x-field-uid: 3
+        prefix:
+          description: >-
+            The prefix of the IPv4 address.
+          type: integer
+          format: uint32
+          minimum: 1
+          maximum: 32
+          default: 32
+          x-field-uid: 4
+    Flow.RSVP.PathExplicitRouteType1FourByteASNumber:
+      description: >-
+        Class = EXPLICIT_ROUTE, Type1 ROUTE_RECORD C-Type = 1
+        Subobject: 4-byte AS number, C-Type: 32
+      type: object
+      properties:
+        l_bit:
+          x-field-pattern:
+            description: >-
+              The L bit is an attribute of the subobject.The L bit is set if the subobject represents a loose hop in the explicit route.
+              If the bit is not set, the subobject represents a strict hop in the explicit route.
+            format: integer
+            length: 1
+            default: 0
+            features: [count]
+          x-field-uid: 1
+        length:
+          description: >-
+            The Length contains the total length of the subobject in bytes,including L ,Type and Length fields.  
+            The Length MUST be atleast 4, and MUST be a multiple of 4.
+          $ref: '#/components/schemas/Flow.RSVPExplicitRoute.Length'
+          x-field-uid: 2
+        reserved:
+          x-field-pattern:
+            description: >-
+              Reserved.
+            format: integer
+            default: 0
+            length: 8
+            features: [count]
+          x-field-uid: 3
+        as_number:
+          description: >-
+            Autonomous System number to be set in the ERO sub-object that this LSP should traverse through.
+            This field is applicable only if the value of 'type' is set to 'as_number'.
+            Note that as per RFC3209, 4-byte AS encoding is not supported.
+          type: integer
+          format: uint32
+          default: 0
+          maximum: 65535
+          x-field-uid: 4
+    Flow.RSVPExplicitRoute.Length:
+      type: object
+      properties:
+        choice:
+          description: auto or configured value.
+          type: string
+          default: auto
+          x-field-uid: 1
+          x-enum:
+            auto:
+              x-field-uid: 1
+            value:
+              x-field-uid: 2
+        auto:
+          description: >-
+            The OTG implementation can provide a system generated value for this property.
```

Comment:
The before OTG can be removed, and also the word "implementation"

Reply:
Done.